esp/src/eclwatch/ClusterProcessesQueryWidget.js

```diff
@@ -85,9 +90,15 @@ define([
                 if (currSel.id === this.id + "_Grid") {
                     this.refreshGrid();
                 } else if (currSel.id === this.legacyClustersProcessesIframeWidget.id && !this.legacyClustersProcessesIframeWidget.initalized) {
-                    this.legacyClustersProcessesIframeWidget.init({
-                        src: ESPRequest.getBaseURL("WsTopology") + "/TpClusterQuery?Type=ROOT"
-                    });
+                    if (!dojoConfig.isContainer) {
```

Comment:
These dojo components all have two changes: (1) disabling the tab. And (2) because they are still reachable programmatically (eg, directly thru the url) setting the content to a generic message rather than loading the iframe, resulting in that exception from ESP being shown